bundles/org.openhab.binding.miio/src/main/resources/OH-INF/config/configGatewayDevices.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="https://openhab.org/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/config-description/v1.0.0 https://openhab.org/schemas/config-description-1.0.0.xsd">
+
+	<config-description uri="thing-type:miio:configGatewayDevices">
+		<parameter name="deviceId" type="text" required="true">
+			<label>Device ID</label>
+			<description>Device ID number for communication (in Hex)</description>
+			<advanced>false</advanced>
+		</parameter>
+		<parameter name="model" type="text" required="false">
+			<label>Device Model String</label>
+			<description>Device model string, used to determine the subtype.</description>
+			<advanced>false</advanced>
+		</parameter>
+		<parameter name="refreshInterval" type="integer" min="0" max="9999" required="false">
+			<label>Refresh Interval</label>
+			<description>Refresh interval for refreshing the data in seconds. (0=disabled)</description>
+			<default>30</default>
+			<advanced>true</advanced>
+		</parameter>
+		<parameter name="timeout" type="integer" min="1000" max="60000" required="false">
+			<label>Timeout</label>
+			<description>Timeout time in milliseconds</description>
+			<default>15000</default>
+			<advanced>true</advanced>
+		</parameter>
+		<parameter name="cloudServer" type="text" required="false">
+			<label>Xiaomi cloud Server (county code)</label>
```

Comment:
Labels are expected to be as short as possible. Guideline is 2-3 words with max 20-25 chars. See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions

Words in labels should be capitalized (except prepositions and so on). See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions